src/main/java/Events.java

```diff
@@ -0,0 +1,13 @@
+public class Events extends Task {
```

Comment:
Would it be more appropriate for the class name to be singular since you're making multiple `Event` objects? (Deadlines class as well)